docs/guides/integration-prebid-server-side.md

```diff
@@ -321,7 +290,9 @@ if (!pbjs.getUserIds().euid) {
 }
 ```
 
->CAUTION: If you configure a user ID by calling `setConfig` (or any similar function) twice, you will need to call `refreshUserIds` for the user ID submodules, to reinitialize their ID values.
+:::caution
+If you configure a user ID by calling `setConfig` (or any similar function) twice, you will need to call `refreshUserIds` for the user ID submodules, to reinitialize their ID values.
+:::
 
 ## Checking the Integration
```

Comment:
@sunnywu a question on this. Content differences between these two:
EUID: https://euid.eu/docs/guides/integration-prebid-client-side#checking-the-integration
UID2: https://unifiedid.com/docs/guides/integration-prebid-client-side#checking-the-integration (better, has more content)
Should EUID be the same?

Reply:
yes EUID should be the same (replace UID2 reference to EUID of course)

Reply:
Thanks a million @sunnywu . Done.